ultravox/data/datasets.py

```diff
@@ -159,6 +159,11 @@ def __post_init__(self):
         if self.audio is not None:
             if self.audio.dtype == np.float64:
                 self.audio = self.audio.astype(np.float32)
+            elif self.audio.dtype == np.int16:
+                # Gradio mp3 file submission, as one example, seems to turn an
+                # mp3 into int16, thus it can be useful to convert to float32 if
+                # we come across this.
+                self.audio = self.audio.astype(np.float32) / np.float32(32768.0)
```

Comment:
I'm wondering if this fix obviates the need for these lines:
https://github.com/fixie-ai/ultravox/blob/main/ultravox/inference/infer.py#L70_L71

Reply:
Yeah, and possibly also in UltravoxProcessor too.

Reply:
Would you prefer I look at removing the two you mention as part of this PR, or separately? Probably separately is better, but asking while I learn your PR preferences.

Reply:
nah, go ahead and land this PR, definitely possible that cleaning up those other lines starts pulling on a thread...

Reply:
Will look at the Infer and Processor bits where we may be able to remove things next.